docs/Maintainer-Guidelines.md

```diff
@@ -105,6 +107,29 @@ is a good opportunity to do it) provided the line itself has some kind
 of modification that is not whitespace in it. But be careful about
 making changes to inline patches—make sure they still apply.
 
+### Adding formulae
+Only one maintainer is necessary to approve and merge the addition of a new formula which passes CI. However, if the formula addition
```

Comment:
Was this line break intentional? Other lines seem to have no length limit.

Reply:
@vitorgalvao Probably not then 😁